llvm/lib/Support/Mustache.cpp

```diff
@@ -681,10 +685,59 @@ void ASTNode::renderChild(const json::Value &Contexts, llvm::raw_ostream &OS) {
     Child->render(Contexts, OS);
 }
 
+void ASTNode::indentTextNode(std::string &Body, size_t Indentation,
+                             bool FinalNode) {
+  std::string spaces(Indentation, ' ');
```

Comment:
```suggestion
  std::string Spaces(Indentation, ' ');
```
The style guide prefers that we capitalize variables, and that they should be nouns. https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly